internal/controller/volsync/vshandler.go

```diff
@@ -881,6 +893,82 @@ func (v *VSHandler) TakePVCOwnership(pvcNamespacedName types.NamespacedName) (bo
 	return true, nil
 }
 
+func (v *VSHandler) prepareForCGFinalSync(rgsNamespacedName, pvcNamespacedName types.NamespacedName) error {
+	log := v.log.WithValues("rgs", rgsNamespacedName)
+
+	log.V(1).Info("Prepare for CG final sync")
+
+	err := v.stopRGSScheduling(rgsNamespacedName, log)
```

Comment:
prepareForFinalSync checks for active jobs before stopping scheduling. Should we follow the same order here, or does it not make a difference?

Reply:
Waiting for VGS completion will do the job. Also, in CG, Ramen manages the scheduling and it is handled in `stopRGSScheduling`.